webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -527,16 +546,16 @@ def _define_style_ensemble(index: int, x_axis: pd.Index) -> Dict:
     return style
 
 
-def _update_corr_index_dict(
+def _calculate_first_observation_index(
```

Comment:
Would that make sense that the name of the function is `_get_first_observation_index`. It seems would be more uniform with the names of other functions and in this function we are actually not calculating, but getting the index...What do you think?

Reply:
sure, done : )